docs/guides/visualization.rst

```diff
@@ -0,0 +1,287 @@
+
+.. code:: python3
+
+    import geopyspark as gps
+    import matplotlib.pyplot as plt
+    
+    from colortools import Color
+    from pyspark import SparkContext
+    
+    %matplotlib inline
+
+.. code:: python3
+
+    !curl -o /tmp/cropped.tif https://s3.amazonaws.com/geopyspark-test/example-files/cropped.tif
+
+.. code:: python3
+
+    conf = gps.geopyspark_conf(master="local[*]", appName="visualization")
+    pysc = SparkContext(conf=conf)
+
+.. code:: python3
+
+    raster_layer = gps.geotiff.get(layer_type=gps.LayerType.SPATIAL, uri="/tmp/cropped.tif")
+    tiled_layer = raster_layer.tile_to_layout(layout=gps.GlobalLayout(), target_crs=3857)
+
+
+.. _visualizing:
+
+Visualizing Data in GeoPySpark
+==============================
+
+Data is visualized in GeoPySpark by running a server which allows it to
+be viewed in an interactive way. Before putting the data on the server,
+however, it must first be formatted and colored. This guide seeks to go
+over the steps needed to create a visualization server in GeoPySpark.
+
+.. _pyramid:
+
+Pyramid
+-------
+
+The ``pyramid`` class represents a list of ``TiledRasterLayer``\ s that
+represent the same area where each layer is a level within the pyramid
+at a specific zoom level. Thus, as one moves up the pyramid (starting a
+level 0), the image will become more zoomed by a power of 2 for each
```

Comment:
`more zoomed` It would be better to be more precise here: "the pixel resolution will increase by power of two"